packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,211 @@
+
+#### Supported features
+
+In this section of your guide, list the features that your application supports. For example:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* SLO (Single Log Out)
+* Force authentication
+* Create users
+* Update user attributes
+* Deactivate users
+* Import users
+* Import groups
+* Sync password
+* Profile sourcing
+
+Also include any restrictions. For example:
+
+Okta can't update user attributes for Admin users. This is an API limitation.
+
+> **Note:** You can briefly describe what each feature does. See the SCIM guides from [Configuration guide examples](#configuration-guide-examples).
+
+#### Procedure
+
+This section constitutes the majority of your guide and explains all the configuration steps needed to get your customers set up with your integration. Detail all settings and include any screenshots that can assist the user.
+
+Also include any best practices for your procedure, such as SCIM guidance on setting mappings for attributes, especially required attributes that don't have a default mapping. For example:
+
+> **Note:** The External ID is a required attribute, but it doesn't have a default mapping. This is because some customers prefer to set it to `EmployeeNumber`, and others like to set it to `emailAddress`. Assign the mapping to the correct value for your organization.
+
+#### Troubleshooting and tips
+
+Include this section if there are known issues that apply to the entire configuration. You can include best practices with the step-by-step procedure instructions. You can also include information on how to contact your organization if the customer has any support queries.
+
+### Configuration guide examples
+
+OIDC:
+
+* [Cerby](https://docs.google.com/document/d/e/2PACX-1vRiswyNyRPVYfEMEwPsbMO8Qn11BjAO-FfUsWBBit_IYe88tzQCHTmMPtmF8uPV044HmXQR13adj3LO/pub)
+* [Upwave](https://help.upwave.io/en/articles/4129778-okta-configuration-guide)
+
+SAML:
+
+* [GitHub Enterprise](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Github_Enterprise.html)
+* [Runscope](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Runscope.html)
+* [Salesforce](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html)
+* [Zoom.us](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Zoom.us.html)
+
+SCIM:
+
+* [Atlassian Cloud](https://confluence.atlassian.com/cloud/configure-user-provisioning-with-okta-957492956.html)
+* [Contentful](https://www.contentful.com/help/okta-user-provisioning-integration-with-scim/)
+* [Zscaler](https://help.zscaler.com/zia/saml-scim-configuration-guide-okta)
+
+## OIN limitations
+
+Integrations with the following Okta features can't be published in the OIN catalog:
+
+* **SWA apps:** SWA integrations are no longer accepted for publication in the OIN catalog. However, existing SWA integrations are still maintained by the OIN team.
```

Comment:
I would also note to contact the OIN team if there needs to be update on their SWA app since they can no longer self-serve in OIN manager

Reply:
Added a sentence for this info. Please review.